src/library_pipefs.js

```diff
@@ -0,0 +1,208 @@
+mergeInto(LibraryManager.library, {
+  $PIPEFS__postset: '__ATINIT__.push(function() { PIPEFS.root = FS.mount(PIPEFS, {}, null); });',
+  $PIPEFS__deps: ['$FS'],
+  $PIPEFS: {
+    BUCKET_BUFFER_SIZE: 1024 * 8, // 8KiB Buffer
+    mount: function (mount) {
+      return FS.createNode(null, '/', {{{ cDefine('S_IFDIR') }}} | 511 /* 0777 */, 0);
```

Comment:
Oh right, missed that line. Yeah, that is a bit odd since it creates multiple nodes that have the identical path `/`. Would instead here `return FS.lookupPath('/');` work instead of `return FS.createNode(...);`? That would avoid creating duplicate root nodes. If it doesn't work for some reason, then that's ok if we document this peculiar behavior to understand why. (I know we do have the same proprty in the existing SOCKFS, would be good to figure that out too)